src/app/components/search/SearchResults.module.css

```diff
@@ -26,6 +26,7 @@
 
   .searchHeaderSubtitle {
     @mixin typography-caption;
+    display: flex;
     margin: 0 0 1px;
```

Comment:
I think this is the margin to increase to `5px` from `1px` that I mentioned in my review comment @danielevalverde